packages/node_modules/samples/browser-meetings/index.html

```diff
@@ -31,7 +31,29 @@
       }
 
       video {
-        height: 20vh;
+        height: 30vh;
+        width: 32vw;
```

Comment:
Wanted to confirm this doesn't cause portrait-oriented feed stretching.

Reply:
This will not stretch the video feed but rather the div container that holds it. 

### Landscape mode
<img width="1037" alt="landscape_mode" src="https://user-images.githubusercontent.com/65302481/84527522-154de400-acac-11ea-9942-cf914632a133.png">

### Portrait mode
<img width="586" alt="portrait_mode" src="https://user-images.githubusercontent.com/65302481/84527520-14b54d80-acac-11ea-9520-562d3d345aa4.png">